esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -347,14 +382,34 @@ void EsdlServiceImpl::configureTargets(IPropertyTree *cfg, const char *service)
         Owned<IPropertyTreeIterator> iter = m_pServiceMethodTargets->getElements("Target");
         ForEach(*iter)
         {
-            const char *method = iter->query().queryProp("@name");
+            IPropertyTree &methodCfg = iter->query();
+            const char *method = methodCfg.queryProp("@name");
             if (!method || !*method)
                 throw MakeStringException(-1, "ESDL binding - found target method entry without name!");
-            const char *type = iter->query().queryProp("@querytype");
+
+            IPropertyTree * customRequestTransform = nullptr;
+            try
+            {
+                customRequestTransform = methodCfg.queryPropTree("xsdl:CustomRequestTransform");
+            }
+            catch (IPTreeException *e)
+            {
+                StringBuffer msg;
+                e->errorMessage(msg);
+                VStringBuffer errmsg("Encountered error while fetching \"xsdl:CustomRequestTransform\" from service \"%s\" ESDL configuration: %s ", service, msg.str());
+
+                methodCfg.setProp("@errorMessage", errmsg.str());
```

Comment:
clever... although errors in the methodcfg tree don't seem to belong...
Perhaps the new m_customRequestTransformMap could hold a object w/ the error? 
Anyway, are there other types of errors we might want to report here as well?
Should this one be crtErrorMessage?